docs/programming_guide/filters.rst

```diff
@@ -71,6 +71,18 @@ Your subclass of DXOFilter benefits from the features of DXOFilter:
     - Filtering history recording. If a DXO node is processed by your filter, your filter's class name will be appended to the DXO's "filter_history"
     - Auditing. If your filter is applied, a job audit event will be created to record the fact that the filter is applied to data.
 
+DXO Filter Behavior in 1-N Communication
+==========
+Based on the design, when a DXO filter is applied to a DXO object, it modifies the DXO object in place. This is fine when the DXO object is expected to be sent to only one recipient, as in the case of 1-1 communication.
```

Comment:
Reading from the DXOFilter, I think it is not mandatory "it modifies the DXO object in place."

So suggest remove this paragraph or reword it.

Reply:
make sense, updated